gamedata/alldefs_post.lua

```diff
@@ -101,6 +101,8 @@ function UnitDef_Post(name, uDef)
 			uDef.flankingbonusmode = 2
 		end
 	end
+	uDef.flankingbonusmin = 1.0 -- change from default engine value of 0.9
+	uDef.flankingbonusmax = 2.0 -- change from default engine value of 1.9
```

Comment:
These two lines are no-op due to the next two lines overwriting the value. Merge the branch https://github.com/beyond-all-reason/Beyond-All-Reason/tree/flanking-rework, which sets the default values in modrules.lua, and removes the experimental flanking modoptions. These two lines can then be removed from alldefs_post.lua.

Reply:
flanking-rework should now be merged